objects/objGabyoall.object.gmx

```diff
@@ -0,0 +1,183 @@
+<!--This Document is generated by GameMaker, if you edit it by hand then you do so at your own risk!-->
+<object>
+  <spriteName>sprSpineRed</spriteName>
+  <solid>0</solid>
+  <visible>-1</visible>
+  <depth>0</depth>
+  <persistent>0</persistent>
+  <parentName>prtEnemy</parentName>
+  <maskName>&lt;undefined&gt;</maskName>
+  <events>
+    <event eventtype="0" enumb="0">
+      <action>
+        <libid>1</libid>
+        <id>603</id>
+        <kind>7</kind>
+        <userelative>0</userelative>
+        <isquestion>0</isquestion>
+        <useapplyto>-1</useapplyto>
+        <exetype>2</exetype>
+        <functionname></functionname>
+        <codestring></codestring>
+        <whoName>self</whoName>
+        <relative>0</relative>
+        <isnot>0</isnot>
+        <arguments>
+          <argument>
+            <kind>1</kind>
+            <string>//Creation code (all optional):
+//col = &lt;number&gt; (0 = red (default); 1 = orange; 2 = blue)
+//dir = -1/1 (1 = faces right (default); -1 = faces left)
+
+event_inherited();
+
+healthpointsStart = 1;
+healthpoints = healthpointsStart;
+contactDamage = 3;
+reflectProjectiles = false;
+reflectProjectilesRight = false; //Only reflect projectiles that come from the right
+reflectProjectilesLeft = false; //Only reflect projectiles that come from the left
+
+damageBuster = 0;
+damageBusterCharged = 0;
+
+damageMetalBlade = 0;
+
+damagePharaohShot = 1;
+damagePharaohShotCharged = 3;
+
+damageStarCrash = 0;
+
+damageSilverTomahawk = 0;
+damageWindStorm = 0;
+
+//Enemy specific code
+col = 0; //0 = red; 1 = orange; 2 = blue    
+dir = "hor";
+startDir = 1;
+beginStartDir = startDir;
+
+moveTimer = 0;
+moving = false;
+image_speed = 0.2;
+image_index = 0;
+
+alarm[0] = 1;
```

Comment:
I tested this, but it results in the color not being changed if "col" is set to be a different value in the creation code of an enemy instance in the room. The "alarm[0] = 1" is necessary as GMS so that the creation code will work.